src/components/Footer/Footer.js

```diff
@@ -26,8 +39,18 @@ const Footer = ({
   blockStatus,
   loadingVote,
   showScoreButton = false,
-  onPressScore
+  onPressScore,
+  isShowDM = false
 }) => {
+  const {sendMessageDM} = useDMMessage();
+  const [profile] = React.useContext(Context).profile;
+  const [loading, setLoading] = React.useState({
+    loadingDm: false,
+    loadingDmAnon: false
+  });
+  const {createSignChat} = useCreateChat();
+
+  const refSheet = React.useRef();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `useDMMessage` and `useCreateChat` hooks is introduced here. It's important to ensure that these hooks are implemented correctly and that their return values (`sendMessageDM` and `createSignChat`) are used properly. Also, the introduction of a new state variable `loading` with properties `loadingDm` and `loadingDmAnon` should be handled carefully to avoid unnecessary re-renders.



<!-- This is an auto-generated comment by OSS CodeRabbit -->